x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_actions_popopver.scss

```diff
@@ -0,0 +1,9 @@
+.ruleActionsPopover {
+  .euiContextMenuItem:hover {
+    text-decoration: none;
```

Comment:
Is there a reason we are removing this? We typically show the underline on hover.  As shown [here](https://elastic.github.io/eui/#/navigation/context-menu)

Reply:
No, i just copied the styles from the popover in rules_list.
These are removed there intentionally by adding an extra style :)  i thought this was from UX...

Would you like me to remove that one too?

https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/collapsed_item_actions.scss

Reply:
Yes please!  Perhaps the button targeting change to use a className mentioned below can be applied there as well?